### PR TITLE
Change functions and add more comments on install page 

### PR DESCRIPTION
**Pull request by boubkerbribri**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 1.7.7.x
| Description?      | Change functions and add more comments on install page 
| Type?             | refacto
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | no
| How to test?      | Sanity should be good
| Possible impacts? | none

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/23674)

### CODE REVIEW

File: tests/UI/pages/install/index.js [hunk 1]
```diff
@@ -19,19 +19,19 @@ class Install extends CommonPage {
...
     this.languageSelect = '#langList';
 
     // Selectors for step 2
-    this.secondStepPageTitle = 'h2#licenses-agreement';
+    this.licenseAgreementsStepPageTitle = 'h2#licenses-agreement';
     this.termsConditionsCheckbox = '#set_license';
 
     // Selectors for step 3
-    this.thirdStepPageTitle = '#sheet_system h2';
+    this.sustemCompatibilityStepPageTitle = '#sheet_system h2';
```
Comment (Progi1984) [inline tests/UI/pages/install/index.js]:
```suggestion
    this.systemCompatibilityStepPageTitle = '#sheet_system h2';
```

File: tests/UI/pages/install/index.js [hunk 2]
```diff
@@ -61,24 +61,49 @@ class Install extends CommonPage {
...
+      case 'Choose your language':
+        selector = this.chooseLanguageStepPageTitle;
+        break;
+
+      case 'License agreements':
+        selector = this.licenseAgreementsStepPageTitle;
+        break;
+
+      case 'System compatibility':
+        selector = this.sustemCompatibilityStepPageTitle;
```
Comment (Progi1984) [inline tests/UI/pages/install/index.js]:
```suggestion
        selector = this.systemCompatibilityStepPageTitle;
```

**Review by Progi1984 — COMMENTED** [review]
Small feedbacks with a typo

**Review by Progi1984 — APPROVED** [review]

**Review by matthieu-rolland — APPROVED** [review]

**Comment by Progi1984** [comment]
Thanks @boubkerbribri

**Pull request merged**